### PR TITLE
Add justification for recommendations based on Thoth community

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/prescriptions-refresh-job/issues/41

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Review by KPostOffice — COMMENTED** [review]
Hey Frido, just a couple suggestions. Feel free to expand or reword.

File: _j/thoth_community.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Recommendations computed based on Thoth community
+---
+
+The recommendation engine is providing recommendations based on requests done
```
[the remark this answers is not in the document]
Comment (fridex) [reply _j/thoth_community.md]: Thanks 👍🏻

File: _j/thoth_community.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+to the community resolver.
+
+## Issue description
+
+Thoth provides a community resolver that can be used by broad user-base. Users
```
[the remark this answers is not in the document]
Comment (fridex) [reply _j/thoth_community.md]: Thanks, adjusted 👍🏻

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**